.github/workflows/universalnrp-test.yml

```diff
@@ -1,6 +1,10 @@
 name: Universal NRP Test
 
-on: [workflow_dispatch]
+on:
+  push:
```

Comment:
We should probably restrict this a bit. The default `push` event will run on a push to any branch. If the goal is to get this to run on feature branches, `workflow_dispatch` is best. But if the goal is to also run on PRs, this should stay in the CI workflow. 

Can we restrict this a bit?

Reply:
Oups, that one snuck in. Good catch! Intent is to just have workflows dispatch and pr only